### ISSUE TITLE
isPhysicallyContainedIn belongs to more than one entity types

### ISSUE THREAD

**Issue by shoeffner** [ease-crc/soma#132]
When loading SOMA in owlready2, I get the following WARNING:

```
* Owlready2 * WARNING: ObjectProperty http://www.ease-crc.org/ont/SOMA.owl#isPhysicallyContainedIn belongs to more than one entity types: [owl.DatatypeProperty, owl.ObjectProperty, DUL.isLocationOf]; I'm trying to fix it...
* Owlready2 * WARNING: ObjectProperty http://www.ease-crc.org/ont/SOMA.owl#isPhysicallyContainedIn belongs to more than one entity types: [owl.DatatypeProperty, owl.ObjectProperty, DUL.isLocationOf]; I'm trying to fix it...
* Owlready2 * WARNING: ObjectProperty http://www.ease-crc.org/ont/SOMA.owl#isPhysicallyContainedIn belongs to more than one entity types: [owl.DatatypeProperty, owl.ObjectProperty, DUL.isLocationOf]; I'm trying to fix it...
```

**Comment by shoeffner** [comment]
I found out that Protégé also warns about this:
```
WARN  17:46:48  Illegal redeclarations of entities: 
    reuse of entity http://www.ease-crc.org/ont/SOMA.owl#isPhysicallyContainedIn in punning not allowed 

[Declaration(DataProperty(<http://www.ease-crc.org/ont/SOMA.owl#isPhysicallyContainedIn>)), 
 Declaration(ObjectProperty(<http://www.ease-crc.org/ont/SOMA.owl#isPhysicallyContainedIn>))]
```
(whitespace added for readability)

As you can see, it's once defined as an ObjectProperty, and once as a DataProperty.

<img width="681" alt="image" src="https://user-images.githubusercontent.com/1836815/93495525-7e2da380-f90e-11ea-9e0c-0693d5dd2022.png">

**Comment by daniel86** [comment]
fixed in #135 
not sure how this could have happened :/
